Sources/Algorithms/Consumers/RegexConsumer.swift

```diff
@@ -1,106 +1,76 @@
 import Regex
 
+public struct Regex {
```

Comment:
Erm, do you want to be squatting on a public `Regex` type? Any other name you could give?

Reply:
This is honestly just to make the function signatures look good, and I imagined that we'd just replace it by an actual `Regex` type that isn't just a placeholder. But I also don't mind renaming it if this might cause issues.